notebooks/MAST/TESS/interm_tess_prf_retrieve/tess_prf_retrieval.ipynb

```diff
@@ -0,0 +1,852 @@
+    "\n",
+    "There is no package to retrieve or work with the PRFs. So this tutorial starts by defining some functions to make it easier to parse the files and retrieve the PRF we are looking for. Each function is documented, but a quick overview is given below.   \n",
+    "\n",
+    "If you just want to understand the shapes of the PRFs, skip ahead to the section [Obtain PRF at an image location](#obtain_PRF).\n",
+    "\n",
+    "\n",
+    "### Description of the PRF retrieval code\n",
+    "The primary function (`getPrfAtColRowFits`) takes a column, row, ccd, camera and sector and returns a 13x13 image of the best PRF interpolated from the grid of PRFs provided in the files.  It uses the ccd, camera and sector information to determine the name of the relevant PRF file in `pathLookup`. Then `determineClosestTessColRow` uses the requested row and column, finds the 4 nearest positions on the detector for which the mission provided a PRF file. Open those files and pull out the appropriate sub-pixel image. Once the four 13x13 sub-pixel PRF images are extracted, `interpolatePrf` interpolates them to the specfied pixel location. The main function then returns this interpolated PRF image.\n",
+    "\n",
+    "### Required Software\n",
+    "- astropy.io\n",
+    "- numpy\n",
+    "- matplotlib\n",
+    "- astroquery.mast"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "----"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Code to retrieve a TESS PRF image"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.io import fits\n",
+    "import numpy as np\n",
+    "import os\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Determine the name of the PRF based on ccd camera and sector"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def pathLookup(ccd, camera, sector):\n",
+    "    \"\"\"\n",
+    "    Gets the datestring and the subdirectory for the specified PRF. \n",
+    "    The datestring and directory name can be found from the ccd, camera and sector.\n",
+    "    \n",
```

Comment:
Add inputs and returns sections to docstring.

Reply:
Added.